buildSrc/src/main/java/org/elasticsearch/gradle/info/BuildParams.java

```diff
@@ -89,6 +90,10 @@ public static JavaVersion getRuntimeJavaVersion() {
         return value(runtimeJavaVersion);
     }
 
+    public static String getJvmDetails() {
```

Comment:
Can you please open this as a separate change so it goes to master as well? We should probably also name it like "runtime jvm details"?

Reply:
sure thing, makes sense

Reply:
https://github.com/elastic/elasticsearch/pull/61901